### PR TITLE
OVN IC Post merge readiness for cucushift

### PR DESCRIPTION
**Pull request by anuragthehatter**
@openshift/team-sdn-qe This is for cucshift/verification-tests repo
The changes were done based on same description as in 
https://github.com/openshift/openshift-tests-private/pull/10506

Please help checking you test s any on this repo and report feedback. Thanks

### CODE REVIEW

File: features/step_definitions/networking.rb [hunk 1]
```diff
@@ -1031,70 +1031,81 @@
...
+                                             user: admin, quiet: true) { |pod, hash|
+      pod.node_name == leader_node || pod.ip == leader_node
+    }.first
+    # update the cache so we can execute on the pod without specify the name
+    cache_resources leader_pod
+    cb[cb_leader_name] = leader_pod
+    logger.info "cb.#{cb_leader_name}.name = #{leader_pod.name}"
+    logger.info "cb.#{cb_leader_name}.node_name = #{leader_pod.node_name}"
+  else  
+    step "I select a random node's host"
```
Comment (zhaozhanqi) [inline features/step_definitions/networking.rb:1101]: @anuragthehatter  QQ for random node?  I have not check the OVN_IC details yet? just know each worker will have own DB. see if podA is created on workerA,  However here select a random node(workerB), it that mean the podA info can be also find in workerB DB?

Comment (anuragthehatter) [reply features/step_definitions/networking.rb:1101]: Yea so this is still being evaluated as per I commented on https://github.com/openshift/openshift-tests-private/pull/10506#discussion_r1263817853

But for now seems like exisiting usecsaes with this change should work. I will be making more changes as i test further. Thanks

Comment (anuragthehatter) [reply features/step_definitions/networking.rb]: SUbmitted my all fixes. Please check your cases @openshift/team-sdn-qe . Changes seem to be working fine on both 4.13 and 4.14 (pre-IC bot cluster)

**Comment by anuragthehatter** [comment]
Following changes will go via this PR:

```
1. Given I store the ovnkube-master "north" leader pod in the clipboard for "pod" using node "<%= cb.pod_node %>"
```
Above step on 4.14+ will be executed as is referencing specific ovnkube-node for pod resource
for 4.14 and less, "pod" and pod_node var will be ignored so it will default to NB leader or SB leader whatever is in step

```
Given I store the ovnkube-master "north" leader pod in the clipboard
```
Above step on 4.14+ wil store random ovnkube-node pod as their is no north/souht leader concept now
on 4.13 and less, it will work as default like north or south

``` 
When I store the ovnkube-master "south" leader pod in the :original_south_leader clipboard for "raft" using node "<%= cb.south_leader.node_name %>"
```
Above step will run on 4.13 and less only. @rbbratta we need to syn up on tests using such step to discard running on 4.14+

**Comment by anuragthehatter** [comment]
@openshift/team-sdn-qe Suggesting you to run your cases on this repo and help in providing feedback. Thanks

**Comment by anuragthehatter** [comment]
@rbbratta Following use cases under `ovn.feature` won't make sense to run on OVN IC clusters. let me know if I can disable them to run on 4.13+ in this staging PR

```
28936, 26092, 26139, 26089, 26091, 26140
```

@zhaozhanqi I guess the best way would be to remove `@4.14` tag from then post IC merge? That's it?

**Comment by rbbratta** [comment]
We can disable them on 4.14+

**Comment by zhaozhanqi** [comment]
Yes,  agree, we can remove the tag and also version in Polartion

**Comment by anuragthehatter** [comment]
> We can disable them on 4.14+

Thanks for confirming, Ross. oops, yes 4.14+ I meant

**Comment by anuragthehatter** [comment]
@openshift/team-sdn-qe This PR needs to merge asap IC PR #1874 merges next week on Wed. If you have any comments, lets resolve them before that so that we can merge it to prevent most of failures across networking board.
Else leave lgtm here so its mergeable as soon as IC merges. Tahnks

**Comment by anuragthehatter** [comment]
@zhaozhanqi Please help approving this as IC has been merged.

**Pull request merged**